zebROS_ws/src/controller_node/config/zed_objdet/common.yaml

```diff
@@ -53,7 +53,7 @@ pos_tracking:
     save_area_memory_db_on_exit: false                          # save the "known visual features" map when the node is correctly closed to the path indicated by `area_memory_db_path`
     area_memory:                true                            # Enable to detect loop closure
     floor_alignment:            true                            # Enable to automatically calculate camera/floor offset
-    initial_base_pose:          [0.0,0.0,0.0, 0.0,0.0,0.0]      # Initial position of the `base_frame` -> [X, Y, Z, R, P, Y]
+    initial_base_pose:          [0.0,0.0,0.0, 0.0,20.0,0.0]      # Initial position of the `base_frame` -> [X, Y, Z, R, P, Y]
```

Comment:
Unless the camera's physically been moved on the 2022 robot, this should probably be returned back to 0